lms/djangoapps/teams/static/teams/js/collections/topic.js

```diff
@@ -1,7 +1,7 @@
 ;(function (define) {
     'use strict';
-    define(['teams/js/collections/base', 'teams/js/models/topic', 'gettext'],
-        function(BaseCollection, TopicModel, gettext) {
+    define(['teams/js/collections/base', 'teams/js/models/topic', 'gettext', 'underscore'],
```

Comment:
Nit: I like to put the very generic dependencies first (Backbone, Underscore etc).